### PR TITLE
(perf-issues) add additional thresholds for consecutive db detector

### PR DESCRIPTION
**Pull request by DominikB2014**
This adds two additional thresholds that we can fine tune.
"total_duration" - the duration of the all the consecutive db spans (to prevent cases where it isn't worth making the spans parallel)
"span_duration_threshold" - the duration of each of the independant spans (to prevent cases of many tiny spans that can be in parallel)

### CODE REVIEW

File: src/sentry/utils/performance_issues/performance_detection.py [hunk 1]
```diff
@@ -915,14 +917,28 @@ def _add_problem_span(self, span: Span) -> None:
...
+            get_span_duration(span).total_seconds() * 1000
+            > self.settings.get("span_duration_threshold")
+            for span in independent_db_spans
+        )
+        exceeds_total_duration_threshold = exceeds_duration_threshold = self._sum_span_duration(
+            self.consecutive_db_spans
+        ) > self.settings.get("total_duration_threshold")
         exceeds_duration_threshold = self._sum_span_duration(
             independent_db_spans
         ) > self.settings.get("duration_threshold")
```
Comment (gggritso) [inline src/sentry/utils/performance_issues/performance_detection.py:934]: 🤔 what's the significance of checking both the independent span durations and the consecutive span durations?

Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py:934]: Good question, (and if my logic is off here lmk)
In a set of consecutive spans, the independent spans are the ones that we think can be in parallel. If an independent span take almost as much as the consecutive spans, then there isn't really benefit in parallelizing the query. 
Now there is going to be cases with multiple independent spans in a set of consecutive db spans, for this case this logic doesn't really work, so in the future a cost function will be introduced to help evaluate if its "worth" running the spans in parallel.

Comment (gggritso) [reply src/sentry/utils/performance_issues/performance_detection.py:934]: That makes sense! Would you mind adding a comment to explain this? It also might help to give the thresholds more descriptive names that explain their purpose 🙏🏻

**Review by gggritso — APPROVED** [review]
✨

File: src/sentry/utils/performance_issues/performance_detection.py (hunk 1 above)
Comment (DominikB2014) [reply src/sentry/utils/performance_issues/performance_detection.py:934]: Yeah, for sure!

**Pull request merged**